src/core/mac/sub_mac.cpp

```diff
@@ -471,7 +471,7 @@ void SubMac::StartCsmaBackoff(void)
                     kCcaSampleInterval - kCslTransmitTimeAhead)
```

Comment:
I would expect the following:

```suggestion
                    kCcaSampleInterval - kCslTransmitTimeAhead - kRadioHeaderShrDuration)
```

Reason: only if the compound TxDelay time is in the future, we need to use the timer.  In other cases the "Transmit without delay" case should be chosen.

Reply:
Fixed. :-)